kolibri/plugins/learn/assets/src/views/learn-page/index.vue

```diff
@@ -11,7 +11,9 @@
         :is="recommendationDisplay"
         :gen-content-link="genContentLink"
         :filter="false"
-        :contents="trimmedPopular"/>
+        :contents="trimmedPopular"
+        :showContentKindFilter="false"
```

Comment:
looks like there's already a `:filter="false"` prop that's being ignored?

Reply:
Another reason to avoid `<component/>`, it makes refactoring harder. That filter prop was replaced by 2 other props in https://github.com/learningequality/kolibri/commit/fab4a940e62aa19e3d8806755c653d03e63d92fb, but during the refactor I was searching for `<content-card-group-grid`, so I did not make the changes here.